packages/react-vapor/docs/src/components/ComponentPage.tsx

```diff
@@ -41,20 +42,23 @@ const ComponentPage: React.FunctionComponent<ComponentPageProps> = (props) => {
         loadAll().then((all) => setTabs(all.filter(Boolean)));
     }, [componentRootPath]);
     const {name, component} = props;
-    const hasMarkdownTabs = tabs.length > 0;
+    const hasMarkdownTabs = tabs && tabs.length > 0;
     const getTabId = buildTabIdTemplate(name);
     const mapTabConfigToProps = ({tabName}: TabConfig): ITabProps => ({
         id: getTabId(tabName),
         title: tabName,
     });
 
-    const tabProps: ITabProps[] = hasMarkdownTabs && [
-        {id: getTabId('development'), title: component.firstTabLabel || 'Develop'},
-        ...tabs.sort((tabA, tabB) => tabA.order - tabB.order).map(mapTabConfigToProps),
-    ];
+    const tabProps: ITabProps[] = hasMarkdownTabs &&
+        tabs && [

```

Comment:
no need to check for tabs here since `hasMarkdownTabs` will be false if you don't have any tabs

Reply:
you are right ! :+1: